src/Playwright.Tests/BrowserTypeConnectTests.cs

```diff
@@ -80,21 +80,54 @@ public void TearDown()
         }
 
         [PlaywrightTest("browsertype-connect.spec.ts", "should be able to reconnect to a browser")]
-        [Ignore("SKIP WIRE")]
-        public void ShouldBeAbleToConnectToBrowserAsync()
+        public async Task ShouldBeAbleToConnectToBrowserAsync()
```

Comment:
For all these tests, you should take the original ("should be able to reconnect to a browser" in browsertype-connect.spec.ts for this one), and port them here. Your tests don't seem to be checking the same scenarios.

Reply:
Fixed the tests.